content/doc/book/blueocean/pipeline-editor.adoc

```diff
@@ -47,16 +47,94 @@ The editor can be launched via:
 * Does not have full parity with Declarative Pipeline
 * Pipeline re-ordered and comments removed
 
-== Pipeline Settings
+== Navigation bar
+
+The Pipeline Editor includes the <<getting-started#navigation-bar, standard navigation bar>>
+at the top, with a local navigation bar below that.
+The local navigation bar includes:
+
+* *Pipeline Name* - This will include the branch depending or how
+* *Cancel* - Discard changes made to the pipeline.
+* *Save* - Open the <<#save-pipeline-dialog, Save Pipeline Dialog>>.
+
+== Pipeline settings
+
+By default, the right side of editor shows the "Pipeline Settings".
+This pane can be accessed by clicking anywhere in the
+<<#stage-flow, Stageflow editor>> pane
+that is not a Stage or one of the "Add Stage" nodes.
 
 === Agent
 
+The "Agent" section controls what agent the Pipeline will use.
+This is the same as the <<../pipeline/syntax#agent, "agent" directive>>.
+
 === Environment
 
-== Pipeline Flow
+The "Environment" sections lets us set environment variables for the Pipeline.
+This is the same as the <<../pipeline/syntax#environment, "environment" directive>>.
+
+== Stageflow editor
+
+The left side editor screen contains the Stageflow editor,
+used for creating the stages and parallel branches of a Pipeline.
+
+image:blueocean/editor/stageflow-basic.png[Simple Stageflow, role=center]
+
+Stages can be added to the Pipeline by clicking the "+" node to the right of an existing stage.
+Parallel branches can be added to a stage by clickingthe "+" node below an existing Stage.
```

Comment:
Use s/parallel branches/Parallel stages/ instead. This becomes more correct terminology when we merge in the work to make this truely parallel stages so people need to get used to it.